app.js

```diff
@@ -0,0 +1,192 @@
+    const board = document.getElementsByClassName("box");
+    document.getElementById("game-status").innerHTML = "status = running";
+
+    for (let i = 0; i < board.length; i++) {
+      if (game.playCount < 9) {
+        board[i].addEventListener("click", function () {
+          if (board[i].innerHTML.trim() === "" && game.status === "Running") {
+            board[i].innerHTML = game.currentPlay.symbol;
+            if (game.currentPlay.symbol === "X") {
+              game.currentPlay = game.oPlayer;
+            } else {
+              game.currentPlay = game.xPlayer;
+            }
+            document.getElementById("player").innerHTML = game.currentPlay.name;
+
+            if (
+              board[0].innerHTML === board[1].innerHTML &&
+              board[1].innerHTML === board[2].innerHTML &&
+              board[0].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[3].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[5].innerHTML &&
+              board[3].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[6].innerHTML === board[7].innerHTML &&
+              board[7].innerHTML === board[8].innerHTML &&
+              board[6].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[0].innerHTML === board[3].innerHTML &&
+              board[3].innerHTML === board[6].innerHTML &&
+              board[0].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[1].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[7].innerHTML &&
+              board[1].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[2].innerHTML === board[5].innerHTML &&
+              board[5].innerHTML === board[8].innerHTML &&
+              board[2].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[0].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[8].innerHTML &&
+              board[0].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[2].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[6].innerHTML &&
+              board[2].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else {
+              game.playCount++;
+              if (game.playCount === 9) {
+                document.getElementById("reset").classList = "";
+              }
+            }
+          }
+        });
+      }
+    }
+  };
+
+  const showWinner = function (game) {
+    document.getElementById("winner").innerHTML = "with 3 straight symbols";
```

Comment:
It would be nice to have something like this => `Congratulation player name`

![image](https://user-images.githubusercontent.com/31897434/82553386-d2409b00-9b5b-11ea-860e-03f3e5afdeb6.png)
https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript